packages/langchain/src/orchestration/client.ts

```diff
@@ -182,26 +180,57 @@ export class OrchestrationClient extends BaseChatModel<
     );
 
     for await (const chunk of response.stream) {
+      const orchestrationResult = chunk.data.orchestration_result;
+      // There can be only none or one choice inside a chunk
+      const choice = orchestrationResult?.choices[0];
+
+      // Map the chunk to a LangChain message chunk
       const messageChunk = mapOrchestrationChunkToLangChainMessageChunk(chunk);
-      const tokenIndices = computeTokenIndices(chunk);
-      const finishReason = response.getFinishReason();
-      const tokenUsage = response.getTokenUsage();
 
-      setFinishReason(messageChunk, finishReason);
-      setTokenUsage(messageChunk, tokenUsage);
+      // Create initial generation info with token indices
+      const newTokenIndices: NewTokenIndices = {
+        prompt: options.promptIndex ?? 0,
+        completion: choice?.index ?? 0
```

Comment:
[for reviewer] Supporting orchestration multi choice by adding this line. It is simple because each chunk contain only one choice.